### DISCUSSION TITLE
Have ESLint migrator output which rules it could not migrate?

### DISCUSSION THREAD

**Discussion by voxpelli** [biomejs/biome#3282] [Ideas]
Would it be possible to have ESLint migrator output which rules it could not migrate?

I did an experimental migration of neostandard: https://github.com/neostandard/neostandard/issues/77 But without me doing some manual diffs and math I can't tell how much of the neostandard config that the migration covers – whether it is a 40% subset, a 80% subset and which rules (maybe some vital ones) that couldn't be migrated.

If one could even get the migrator to output this in JSON or similar (eg. using a `--json` flag) then I could probably write up a GitHub Action that continuously generates an up to date Biome config for every new change in neostandard.

**Comment by Conaclos** [comment]
An earlier version of the ESLint migrator displayed the rules that were not migrated. However, this was too much noise for the user, I decided to remove it.

However, you made a good point. We could perhaps support something similar to the linter/formatter reporters:

```shell
❯ biome migrate eslint --reporter=json
{
    // format to define
}
```

The reporter could output the migrated rules and the non-migrated rules.

**Reply by voxpelli** [reply]
That would be fantastic 🙏

**Reply by Conaclos** [reply]
I opened a task #3397
